### ISSUE TITLE
[suggestion] Use the Git tags to keep up-to-date with new releases

### ISSUE THREAD

**Issue by nelson6e65** [ondras/wwwsqldesigner#236]
Currently, we are using a master-dev version, and we don't know what actual "version" are we using, or its stability (stable/beta/alpha/**dev**) for updates.

Should be nice to have use of [GitHub releases](https://github.com/ondras/wwwsqldesigner/releases) tags to keep up-to-date with releases of this awesome tool! :smiley:

**Comment by ondras** [comment]
@nelson6e65 yeah, you are correct. That would be great. And systematic.

Unfortunately, there really is no development going on with this app; my time is rather limited and the only code being commited are bugfixes (as well as translations). So until some real "features" start to appear/land (which is unlikely, given my current situation and the fact that I am the sole developer of this thingy), the status quo is "every commit is a bugfix for the one-and-only stable version, which was named 2.7 back in the 2012, but nobody really cares today".

**Comment by nelson6e65** [comment]
:sob:

Well... I'm glad this is still alive. Is an useful tool! :bowtie:

**Comment by ondras** [comment]
Yeah, definitely alive and usable. Just not under active development. **Mature** ;-)
